webots_ros2_control/include/webots_ros2_control/Ros2ControlSystem.hpp

```diff
@@ -54,6 +54,10 @@ namespace webots_ros2_control {
 
     rclcpp_lifecycle::node_interfaces::LifecycleNodeInterface::CallbackReturn on_init(
       const hardware_interface::HardwareInfo &info) override;
+#if HARDWARE_INTERFACE_VERSION_MAJOR >= 5 && HARDWARE_INTERFACE_VERSION_MINOR >= 3
```

Comment:
These tests are error prone, we should either:
```c++
#if HARDWARE_INTERFACE_VERSION_MAJOR == 5 && HARDWARE_INTERFACE_VERSION_MINOR >= 3
```
or
```c++
#if HARDWARE_INTERFACE_VERSION_MAJOR > 5 || (HARDWARE_INTERFACE_VERSION_MAJOR == 5 && HARDWARE_INTERFACE_VERSION_MINOR >= 3)
```
Otherwise, it won't work work versions 6.0, 6.1, 6.2, 7.0, 7.1, etc. but will work with versions 6.3, 6.4, 7.8, etc. which is odd.